src/main/java/edu/harvard/iq/dataverse/DatasetServiceBean.java

```diff
@@ -727,6 +728,27 @@ public void exportAllDatasets(boolean forceReExport) {
 
     }
     
+    //get a string to add to save success message
+    //depends on dataset state and user privleges
+    public String getReminderString(Dataset dataset, boolean canPublishDataset) {
+
+        if( dataset.getId() == null || !dataset.isReleased() ){
```

Comment:
I wonder why we have to check the null here? I know you just moved this from somewhere else, so it's fine if we leave it. That other place had a comment that said "dataset id is null before first save -- needed for create mode", but even in that circumstance wouldn't isReleased still also be false? So seems extraneous.

Reply:
We don't. When I wrote the method originally I had it higher in the Save method so that newly created datasets did not have an id, but the permissions wrapper requires an id before it can determine canPublishDataset so I moved the call to just before rendering the success method. I will remove it here.